sdk/eventhub/azure-eventhub/tests/perfstress_tests/send_event_batch.py

```diff
@@ -0,0 +1,40 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._test_base import _SendTest
+
+from azure_devtools.perfstress_tests import get_random_bytes
+
+from azure.eventhub import EventData
+
+
+class SendEventBatchTest(_SendTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        self.data = get_random_bytes(self.args.event_size)
+
+    def run_sync(self):
+        batch = self.producer.create_batch()
+        for i in range(self.args.num_events):
```

Comment:
use _ here since i is not being used
```suggestion
        for _ in range(self.args.num_events):
```

Reply:
updated :)